### PR TITLE
Implement Features app

### PR DESCRIPTION
**Pull request by cadecairos**
1. Replace opportunities app with features
2. Add publish_after and expires (both dateTimeField types) to features
3. Filter the api views for features based on publish_after and expires

### CODE REVIEW

File: app/networkapi/features/views.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class FeatureListView(ListAPIView):
+    """
+    A view that permits a GET to allow listing of Features
+    in the database
+    """
+    now = timezone.now()
+    queryset = Feature.objects.filter(
+        Q(publish_after__lt=now),
```
Comment (gideonthomas) [inline app/networkapi/features/views.py]: why do you need Q classes for this? You can simply do `publish_after__lt=now`

File: app/networkapi/features/views.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    pagination_class = None
+
+
+class FeatureView(RetrieveAPIView):
+    """
+    A view that permits a GET request for an Feature in the database
+    """
+    now = timezone.now()
+    queryset = Feature.objects.filter(
+        Q(publish_after__lt=now),
```
Comment (gideonthomas) [inline app/networkapi/features/views.py]: same question

File: app/networkapi/features/views.py (hunk 1 above)
Comment (cadecairos) [inline app/networkapi/features/views.py]: I can make it a kwarg, but I'll have to pass it as the second param. I thought it read better this way.

Comment (gideonthomas) [inline app/networkapi/features/views.py]: I'll approve the change. I'm not super particular but in general Q classes are meant for complex queries that aren't supported by vanilla django querysets.

Comment (cadecairos) [inline app/networkapi/features/views.py]: cool. either way, it all just gets converted to SQL!

**Pull request merged**